bundles/CoreBundle/DependencyInjection/Configuration.php

```diff
@@ -2276,4 +2277,36 @@ private function addGlossaryNode(ArrayNodeDefinition $rootNode): void
                             ->useAttributeAsKey('name')
                             ->prototype('scalar');
     }
+    
+    /**
+     * @param ArrayNodeDefinition $rootNode
+     */
+    private function addNotificationsNode(ArrayNodeDefinition $rootNode): void
```

Comment:
I think we should probably put this into the `AdminBundle` where it belongs to.

Reply:
updated.